### ISSUE TITLE
TEAMMATES logo not transparent

### ISSUE THREAD

**Issue by whipermr5** [TEAMMATES/teammates#6747]
The black background is obvious against the new responsive navbar which has a gradient fill. This is the current view of the front page:

<img width="362" alt="screen shot 2017-03-07 at 12 20 12 am" src="https://cloud.githubusercontent.com/assets/6811428/23618677/0c8c5564-02cc-11e7-8507-0205eaf24d1f.png">

**Comment by li-kai** [comment]
@damithc Is there a photoshop file or higher res version of the logo?

**Comment by damithc** [comment]
Unfortunately, no. This was done by a friend of the first TEAMMATES developer and we only have this one single image file. Time to create a new logo? :-p

**Comment by Dartteon** [comment]
![teammates](https://cloud.githubusercontent.com/assets/9067940/23667011/85ff651e-0397-11e7-8843-a0f883989817.png)
I tried recreating the logo with Illustrator, but could not find the exact font as the original. (I tried parsing it through online font identifiers, to no avail)

I also tried to Image Trace the original logo in Illustrator but it was in too small a resolution to give an accurate trace.

Also had fun designing other versions of the logo, see if you like any of them :P

**Comment by damithc** [comment]
@Dartteon thanks for giving this a try.
For now, we can try to recreate the logo to match the original version -- there is no need to match the exact font though. 
A 'complete redesign' of the logo can be considered at a later stage.

**Comment by Dartteon** [comment]
@damithc Sure thing. Which of the samples do you think matches the original best? I'll do a cleanup and resize for it.

**Comment by samsontmr** [comment]
@Dartteon @damithc I personally like the first one :P

**Comment by damithc** [comment]
> I personally like the first one

So do I. It has similar dimensions to the original.

**Comment by Dartteon** [comment]
Shall I clean the image up, then? Also, do you want the kerning (spacing) between the two Ms to be smaller, like the original?

I personally find that having the two SMS joined together makes it hard to read.

**Comment by samsontmr** [comment]
No I think it looks better the way you did it.

**Comment by damithc** [comment]
> Also, do you want the kerning (spacing) between the two Ms to be smaller, like the original?

The current way is fine.

**Comment by Dartteon** [comment]
![image](https://cloud.githubusercontent.com/assets/9067940/23701529/21651356-0432-11e7-9329-9db2cc04e90f.png)
Shall I export this? Do I make a pull request for image changes too?

**Comment by damithc** [comment]
> Shall I export this? Do I make a pull request for image changes too?

@Dartteon how about you include the logo in a PR that fixes this issue?

**Comment by Dartteon** [comment]
@damithc I am facing a few problems here -

1) To prevent the background color being visible, the image must be a PNG. Currently, all the TEAMMATES logo images are JPEG files, which cannot handle transparency.

2) I'm working on another issue, #6746. The fix for that issue is to change all the `.tag` pages to use the TEAMMATES image.

3) If I change the current TEAMMATES image to PNG, I must save all the images as a PNG. It would then conflict with the fix for (2), which is to use the current JPEG images.

Is it possible to merge the two tickets together? Or should I resolve the issues one at a time? And if the latter, which issue should be resolved first?

**Comment by whipermr5** [comment]
Resolve one at a time :) #6746 is of higher priority so you might want to complete that one first. For that PR, you might also want to look into referencing the image file only once, perhaps in another `.tag` that the other `.tag`s refer to. Then in your PR for this issue, you would just include the new `.png` and the updated path to the image file in the single `.tag` file.
